Makefile

```diff
@@ -0,0 +1,19 @@
+default: test
+
+install:
+	docker-compose run --rm tests npm install
+
+update:
+	docker-compose run --rm tests npm update
+	docker-compose run --rm tests npm update
```

Comment:
Is it necessary to run `npm update` twice in a row? If so, maybe a comment is in order. FWIW, I googled "run npm update twice" and only found a (rather old) Github issue that was closed. Also, I think I saw something the other day where a second run updated more dependencies. Seems like a defect in npm to me.